src/System.Devices.Gpio/System.Devices.Gpio.Samples/Program.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System.Diagnostics;
+using System.Threading;
+
+namespace System.Devices.Gpio.Samples
+{
+    internal class Program
+    {
+        private static void Main(string[] args)
+        {
+            try
+            {
+                var option = -1;
+
+                if (args.Length > 0)
+                {
+                    option = Convert.ToInt32(args[0]);
+                }
+
+                switch (option)
+                {
+                    case 0: Unix_BlinkingLED(); break;
```

Comment:
The Unix cases are using the GpioPin with UnixDriver (general driver using file interface).
The UnixDriver cases are using directly the UnixDriver (no GpioPin usage).
The RaspberryPi cases are using the GpioPin with RaspberryPiDriver (specific driver for that device).
The RaspberryPiDriver cases are using directly the RaspberryPiDriver (no GpioPin usage).

I would like to have the driver internal so users have to use GpioPin class, but also allow them to extend the API with their own drivers implementation some how. I am not sure how this will be possible yet.

Reply:
perhaps rename Unix to "UnixGeneric" or something then.  It's not obvious from the names how they're different.